usagov/vote-gov-nvrf-app
# VOTE-2704: Data analytics for miscellaneous links

Ticket: https://cm-jira.usa.gov/browse/VOTE-2704

Testing steps:
1. Go through the form locally until the "Identification" page
2. Select "State driver ID" in the selector
3. In the console, type 'dataLayer' and press Enter
4. Confirm this value is in the dataLayer: {NVRF_id_type: 'driver-id-num', event: 'NVRF_SELECT_ID'}
5. In the markup, confirm the Progress bar (each step) and the back buttons have the correct data-test attributes (these have no function in the app, they are just used in GA)
